src/tarballs/config.ts

```diff
@@ -29,12 +29,25 @@ export type UpdateConfig = Interfaces.OclifConfiguration['update'] & {
   s3?: Interfaces.S3 & {acl?: ObjectCannedACL}
 }
 
+export type PackConfig = {
```

Comment:
@mdonnalley does this look good? I may have some missing context here, is this config definition supposed to go in the core package?